tyk-headless/templates/configmap-tyk-k8s.yaml

```diff
@@ -13,7 +13,7 @@ data:
       watchNamespaces:
 {{ toYaml .Values.tyk_k8s.watchNamespaces | indent 8 }}
     Tyk:
-      url: "https://localhost:9696"
+      url: "http{{- if .Values.gateway.tls }}s{{- end }}://localhost:9696"
```

Comment:
If we are trying to make this configurable, this port should probably not be hardcoded.

Reply:
good point, used `{{ .Values.gateway.api.containerPort }}`